lib/zold/commands/remote.rb

```diff
@@ -89,6 +91,21 @@ def run(args = [])
         o.bool '--reboot',
           'Exit if any node reports version higher than we have',
           default: false
+        # @todo #292 Group options by subcommands
+        #  Having all the options in one place _rather than grouping them by subcommands_
+        #  makes the help totally misleading and hard to read.
+        #
```

Comment:
@botanicus Don't use empty lines, they are code smell: https://www.yegor256.com/2014/11/03/empty-line-code-smell.html

Reply:
That’s a very interesting opinion. What you say makes sense for the code, but does it really apply here?

Because – and as you know I’m totally new to PDD – this is basically body of a new ticket.

I’m of course happy to adjust to whatever, but I’d like to suggest that this applies to code, not to comments (= documentation).

Let me know please.